BY_LAWS.md

```diff
@@ -1,69 +1,254 @@
-# Thunderbird Council By-Laws
+# Thunderbird Council Bylaws
 
-## Thunderbird Council Election Process
+## Table of Contents
+
+1. [Definitions](#definitions)
+2. [Mandate](#mandate)
+3. [Election Procedure](#election-procedure)
+4. [Membership](#membership)
+5. [Thunderbird Council](#thunderbird-council)
+6. [Call for re-election](#call-for-re-election)
+7. [Dissolution](#dissolution)
+8. [Meetings](#meetings)
+9. [Decision-Making Process](#decision-making-process)
+10. [Confidentiality](#confidentiality)
+11. [Code of Conduct](#code-of-conduct)
+12. [Amendments to these bylaws](#amendments-to-these-bylaws)
+13. [Separability of Articles](#separability-of-articles)
+
+## Definitions
```

Comment:
We had talked about numbering each section, we can do this manually if we want but isn't quite as important as getting the text out.